app/assets/stylesheets/custom.scss

```diff
@@ -1,29 +1,35 @@
 .home-search {
+  text-align: center;
+  background:rgba(0,0,0,0.5);
   vertical-align:middle;
   margin-bottom:0px;
   margin-top: 0px;
-  width: 700px;
+  width: 100%;
+  min-height: 90px;
   margin: 0 auto;
+  padding: 2%;
   position: relative;
-  top: -80px;
+  top: -100px;
   z-index: 1001;
 }
 
 .navbar-header {
+  color: #fff !important;
   margin-top: 10px;
   margin-left: 110px;
   font-size: 21px;
 }
 
-.navbar.transparent .navbar-inner {
+.navbar-inner {
+  color: #fff !important;
```

Comment:
Properties should be ordered background, color, left, margin-left, margin-right, position, right, top, z-index